### DISCUSSION TITLE
Release assets descriptions

### DISCUSSION THREAD

**Discussion by paolo2504** [GrandOrgue/grandorgue#962] [Developers]
I ask if possible to describe the OS which assets of GO releases belong in the download page.
I want to download releases for raspberry  and ubuntu , both 32bit, but i see many files and i am not sure which is the right one.
thanks
note: i recently installed r0131os_armhf.deb with .demo and .resources on my raspberry 3b+ ... Now i don't find these three files available with recent release. I also suppose that corrent release for ubuntu should be ...linux.deb but only 64bit version can be downloaded...

**Comment by oleg68** [comment]
@paolo2504 please take a look at [INSTALL.md](https://github.com/GrandOrgue/grandorgue/blob/master/INSTALL.md)

**Reply by paolo2504** [reply]
Thanks Oleg. I first tried linux armhf.tar... so unpacked in folder ... Launching bin/GrandOrgue i get error msg "bash: GrandOrgue: comand not found" so i am stuck for the moment on this.
I'll give a try with .deb installation

**Reply by paolo2504** [reply]
ok GO 3501 deb package installed and working (on raspberry pi3b+).
Tomorrow i'll give a try on a machine with Lubuntu.

note: launching GrandOrgTool i get it working (so i'll try it to compare with wavpack under Windows...).
However some warning and critical msg arise (see below).


![2022-01-15-003234_1360x768_scrot](https://user-images.githubusercontent.com/50132661/149598784-1786e8e9-ff75-4169-8a1f-98e1bad4c58a.png)

**Reply by paolo2504** [reply]
The above error msg (the last for errors on glib) come out even typing grandorguetool -h for help. Perhaps is related to raspbian ?
I found this on the net:

https://stackoverflow.com/questions/1983427/what-does-glib-gobject-warning-instance-with-invalid-null-class-pointer

**Reply by paolo2504** [reply]
> 

I'm trying with armh.deb on pc running ubuntu mate 32bit and intel processor: no way, it's saying architecure not supported using grandorgue_3.5.0-1_armhf.deb.
If i use grandorgue-3.5.0-1.linux.armhf.tar.gz when, after unpacking, i runGrandOrgue it says "command not found" from terminal.
What am i missing pls ?

Note : processor is Intel pentium 32bit (one of those early model capable to partially support 64bit os.  Ubuntu mate is 18.04 32 bit. 
Probably I need a 32 bit GO package for intel cpu,  isn’t it ?

Also note that the same package …armholes.deb was fine for raspberry pi3b+ (64bit p I) with raspbian/rpios 32bit).

**Reply by paolo2504** [reply]
note: when trying to execute the unpacked release grandorgue-3.5.0-1.linux.armhf.tar.gz i use the syntax:
./GrandOrgue after cd /<unpackedpath>/bin directory   (this way i get cannot execute binary file: executable format not valid)
or with sudo:
sudo ./GrandOrgue,  (this time i get Syntax error: word unexpected (expecting ")")

**Reply by paolo2504** [reply]
From the GUI is not possible to run bin/GrandOrgue: no way.
So i tried once again the deb package grandorgue_3.5.0-1_armhf.deb but launching it from the GUI the deb installer says: "error, wrong architecture armhf; launch dpkg -- add-architecture...." but this leads to other errors and i am not sure i want to do so.
Lastly i tried from terminal with command sudo apt install ./grandorgue_3.5.0-1_armhf.deb.
This time installation starts but suddenly stops with errors as follows.
I may suppose those dependencies are missing but it also says they cant be installed or are blocked.
Pls what to do ? in install.md it is written just unpack and run bin/grandorgue. Sounds easy...
Or is it definitely possible that i need 32 bit grandorgue deb package ??
thanks for anyone hint.



I seguenti pacchetti hanno dipendenze non soddisfatte:
 grandorgue:armhf : Dipende: libasound2:armhf (>= 1.0.16) ma non è installabile
                    Dipende: libc6:armhf (>= 2.15) ma non è installabile
                    Dipende: libfftw3-single3:armhf (>= 3.3.5) ma non è installabile
                    Dipende: libgcc1:armhf (>= 1:3.0) ma non è installabile
                    Dipende: libjack-jackd2-0:armhf (>= 1.9.10+20150825) ma non è installabile oppure
                             libjack-0.125:armhf ma non è installabile
                    Dipende: libstdc++6:armhf (>= 6) ma non è installabile
                    Dipende: libwavpack1:armhf (>= 4.40.0) ma non è installabile
                    Dipende: libwxbase3.0-0v5:armhf (>= 3.0.4+dfsg) ma non è installabile
                    Dipende: libwxgtk3.0-gtk3-0v5:armhf (>= 3.0.4+dfsg) ma non è installabile
                    Dipende: zlib1g:armhf (>= 1:1.1.4) ma non è installabile
E: Impossibile correggere i problemi, ci sono pacchetti danneggiati bloccati.

**Reply by oleg68** [reply]
> Probably I need a 32 bit GO package for intel cpu, isn’t it ?

We do not provide it now. Now only 64bit is supported for intel.

**Reply by larspalo** [reply]
But you can still compile it yourself, I guess. Personally, I've not used 32 bit Linux since a long time, now except on Raspberry Pi, so I haven't tested building GO on 32 bit for many years... Check the BUILD file for instructions if you need them.
